test/fixtures/users.yml

```diff
@@ -25,6 +25,7 @@ rolf:
   name: Rolf Singer
   license: ccwiki30
   image: rolf_profile_image
+  image_size: <%= User.image_sizes[:medium] %>
```

Comment:
Repeats the default in order to make it less likely that a coder will change the value.